src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -1432,6 +1479,14 @@ export class Workbench extends Disposable implements IPartService {
 		this.workbenchLayout.layout();
 	}
 
+	setMenubarHidden(visibility: string, skipLayout: boolean): void {
```

Comment:
@sbatten maybe visibility could have a better type to reflect the values this can have (e.g. `visibility: 'visible' | 'default'`)

Reply:
how do I then pass a string from the configuration to it? I union'd with string for the time being